### PR TITLE
closeio-hipchat addon. Heroku version

### PR DESCRIPTION
**Pull request by congocongo**
@philfreo accept pls

### CODE REVIEW

File: Procfile [hunk 1]
```diff
@@ -0,0 +1 @@
+web: python main.py
```
Comment (philfreo) [inline Procfile]: let's use Gunicorn in production

File: requirements.txt [hunk 2]
```diff
@@ -0,0 +1,6 @@
+SQLAlchemy==0.9.8
+psycopg2==2.4.5
+Flask-SQLAlchemy==2.0
+Flask-Cache==0.13.1
+-e git://github.com/congocongo/flask-hipchat-addon.git#egg=flask-hipchat-addon
```
Comment (philfreo) [inline requirements.txt]: please freeze this to a specific commit hash

File: settings.py [hunk 3]
```diff
@@ -0,0 +1,12 @@
+#SQLALCHEMY_DATABASE_URI = 'sqlite:////tmp/test.db'
+
+HIPCHAT_ADDON_KEY = 'closeio-hipchat'
+HIPCHAT_ADDON_NAME = 'close.io'
+HIPCHAT_ADDON_DESCRIPTION = 'A HipChat add-on to give details about a Close.io lead when its URL is mentioned in HipChat'
+HIPCHAT_ADDON_VENDOR_URL = 'http://close.io'
+HIPCHAT_ADDON_VENDOR_NAME = 'close.io'
```
Comment (philfreo) [inline settings.py]: for vendor and add-on name, use `Close.io` (capital C) if that works.

File: templates/configure.html [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+{% endif %}
+<section id="content" role="main">
+  <!-- Content -->
+  <div class="aui-page-panel">
+    <div class="aui-page-panel-inner">
+      <section class="aui-page-panel-content">
+        <div>
+          <form class="aui unsectioned" id="addon-config-form" method="post">
+            <div class="field-group">
+              <label for="apikey">close.io API key<span class="aui-icon icon-required">required</span></label>
```
Comment (philfreo) [inline templates/configure.html]: Close.io

File: templates/lead.html [hunk 5]
```diff
@@ -0,0 +1,12 @@
+<b>{{lead.display_name}}</b><br/>
```
Comment (philfreo) [inline templates/lead.html]: can you go ahead and also make this a hyperlink to the lead?

**Comment by philfreo** [comment]
Please include a README file with instructions on how to create & deploy the heroku app with all the requirements (from the command line)

File: templates/lead.html [hunk 6]
```diff
@@ -0,0 +1,12 @@
...
+{% if lead.contacts %}
+  <b>Contacts:</b> {% if lead.contacts[0].name %}{{lead.contacts[0].name}}{% else %}Unnamed{% endif %}
+  {% if lead.contacts|length > 1 %}+{{ lead.contacts|length - 1}}{% endif %}<br/>
+{% endif %}
+<b>Status:</b> {{lead.status_label}}<br/>
+{% if lead.opportunities %}
+<b>Opportunities:</b>
+  {# {{organization.currency_symbol}} #} {{lead.opportunities[0].value}}
```
Comment (philfreo) [inline templates/lead.html]: currency symbol needs to be added/working

Comment (philfreo) [inline templates/lead.html]: value needs to be divided by 100.0

File: templates/lead.html [hunk 7]
```diff
@@ -0,0 +1,12 @@
+<b>{{lead.display_name}}</b><br/>
+{% if lead.contacts %}
+  <b>Contacts:</b> {% if lead.contacts[0].name %}{{lead.contacts[0].name}}{% else %}Unnamed{% endif %}
+  {% if lead.contacts|length > 1 %}+{{ lead.contacts|length - 1}}{% endif %}<br/>
+{% endif %}
+<b>Status:</b> {{lead.status_label}}<br/>
+{% if lead.opportunities %}
+<b>Opportunities:</b>
```
Comment (philfreo) [inline templates/lead.html:8]: you can just drop this label altogether. it's obvious by the word 'opportunity' below what it is

File: templates/configure.html [hunk 8]
```diff
@@ -0,0 +1,72 @@
...
+                <input id="apikey" name="apikey"  class="text long-field" title="API key" value="{{ api_key }}"/>
+            </div>
+            <div>
+              <button class="aui-button" type="submit">Save</button>
```
Comment (philfreo) [inline templates/configure.html:64]: Please also provide the following text on this configure page:

You can find your API key in Close.io under Setting > Your API Keys.

File: templates/lead.html (hunk 7 above)
Comment (congocongo) [inline templates/lead.html:8]: I solved it like contacts

**Comment by congocongo** [comment]
@philfreo 
review it again

File: LICENSE [hunk 9]
```diff
@@ -1,6 +1,6 @@
 The MIT License (MIT)
 
-Copyright (c) 2015 Phil Freo
+Copyright (c) 2015 Alexander Deynichenko
```
Comment (philfreo) [inline LICENSE]: this was work made for hire - copyright needs to be assigned to Elastic Inc

File: settings.py [hunk 10]
```diff
@@ -0,0 +1,12 @@
+#SQLALCHEMY_DATABASE_URI = 'sqlite:////tmp/test.db'
+
+#HIPCHAT_ADDON_KEY = 'closeio-hipchat'
+#HIPCHAT_ADDON_NAME = 'Close.io'
+#HIPCHAT_ADDON_DESCRIPTION = 'A HipChat add-on to give details about a Close.io lead when its URL is mentioned in HipChat'
```
Comment (philfreo) [inline settings.py]: some of these should be uncommented right?

**Comment by congocongo** [comment]
@philfreo and again

**Pull request closed without merging**